docs/src/main/tut/applicative.md

```diff
@@ -7,3 +7,33 @@ scaladoc: "#cats.Applicative"
 ---
 # Applicative
 
+Applicative functors are a simple extension of the [Apply
+functor](apply.html) which adds a single method, `pure`:
+
+```scala
+    def pure[A](x: A): F[A]
+````
+
+This method takes any value and returns the value in the context of
+the functor. For many familiar functors, how to do this is
+obvious. For Option, the `pure` operation wraps the value in
+`Some`. For `List`, the `pure` operation returns a single element
+`List`:
+
+```tut
+import cats._
+import cats.std.all._
+
+Applicative[Option].pure(1)
+Applicative[List].pure(1)
+```
+
+Like [Functor](functor.html) and [Apply](apply.html), Applicative
+functors also composes naturally with other Applicative functors, when
```

Comment:
Minor nitpick, but I think this sentence/paragraph would read more easily if you changed this from `, when` to `. When`.

:+1: thanks for all the great docs!